programs/bubblegum/program/src/error.rs

```diff
@@ -86,6 +86,8 @@ pub enum BubblegumError {
     CreatorDidNotUnverify,
     #[msg("Only NonFungible standard is supported")]
     InvalidTokenStandard,
+    #[msg("Canopy size should be set bigger for this tree")]
+    CanopySizeMismatch,
```

Comment:
Maybe we could call this `InvalidCanopySize` as its not really mismatched, its more just out of spec.  What do you think?